express-intro/server.js

```diff
@@ -0,0 +1,6 @@
+import express from 'express';
+const app = express();
+app.listen(8080, () => {
+  // eslint-disable-next-line no-console
```

Comment:
Are you getting a lint warning? If so, open your .eslintrc and remove the rule that says no-console (or change its value to "off").